cmd/showbudget/show_budget.go

```diff
@@ -69,19 +75,43 @@ func (c *showBudgetCommand) Run(ctx *cmd.Context) error {
 	if err != nil {
 		return errors.Annotate(err, "failed to create an http client")
 	}
-	api, err := newAPIClient(client)
+	api, err := newBudgetAPIClient(client)
 	if err != nil {
 		return errors.Annotate(err, "failed to create an api client")
 	}
 	budget, err := api.GetBudget(c.budget)
 	if err != nil {
 		return errors.Annotate(err, "failed to retrieve the budget")
 	}
+	c.resolveModelNames(budget)
```

Comment:
I would have logged the error from resolveModelNames here, but that works too, I guess.